components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1314,16 +1314,16 @@ interface JsonRpcService {
   GetERC1155TokenBalance(string contract_address, string token_id, string account_address, string chain_id) => (string balance, ProviderError error, string error_message);
 
   // Fetches symbol of an ERC20 or ERC721 token.
-  GetEthTokenSymbol(string chain_id,
-                    string contract_address) => (string symbol,
-                                                 ProviderError error,
-                                                 string error_message);
+  GetEthTokenSymbol(string contract_address,
+                    string chain_id) => (string symbol,
+                                         ProviderError error,
+                                         string error_message);
 
   // Fetches decimals of an ERC20 or ERC721 token.
-  GetEthTokenDecimals(string chain_id,
-                      string contract_address) => (string decimals,
-                                                   ProviderError error,
-                                                   string error_message);
+  GetEthTokenDecimals(string contract_address,
+                      string chain_id) => (string decimals,
+                                           ProviderError error,
+                                           string error_message);
   // Solana JSON RPCs
```

Comment:
Could you please double check that these are not called by iOS and Android. Both args are string so it won't break compilation but fail at runtime.

Reply:
Checked that these methods are not used in Android or iOS.